db/migrate/20160418141210_add_read_only_to_miq_alert.rb

```diff
@@ -0,0 +1,14 @@
+class AddReadOnlyToMiqAlert < ActiveRecord::Migration[5.0]
+  def up
+    say_with_time('Add read only parameter to MiqAlert with true for OOTB alerts') do
+      add_column :miq_alerts, :read_only, :boolean, :default => false
```

Comment:
Cab you put the add_column *outside* of the say_with_time?

Additionally, don't put a column level default, as the default for a column is nil which is false anyway (if you really need a `false` prefer using `default_value_for` on the model.